pkg/network/node/vnids.go

```diff
@@ -175,9 +189,27 @@ func (vmap *nodeVNIDMap) Start() error {
 	}
 
 	go utilwait.Forever(vmap.watchNetNamespaces, 0)
+	go utilwait.Forever(vmap.cleanNotFoundMap, 0)
 	return nil
 }
 
+// cleanNotFoundMap() removes any VNID-not-found timestamps that are older than 10 minutes
+// to ensure we don't leave stale entries in the map that will never be found
```

Comment:
This means that if the notfound latency goes beyond 10 minutes, we end up reporting the latency mod 10. (And wasn't it getting up to at least 8 minutes before?)

Stale entries should be pretty rare. We can afford to do the cleanup much less frequently